doc/metrics.md

```diff
@@ -0,0 +1,559 @@
+which is supported by the Opentelemetry Collector. The former is based on the [OpenMetrics
+format](https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md) can be consumed by Prometheus and Thanos or other OpenMetrics compatible
+backends.
+
+_Note_: Both Opentelemetry Javascript and Opentelemetry Collector support
+exporters for different formats, such as [Cloud Monitoring](https://github.com/GoogleCloudPlatform/opentelemetry-operations-js/tree/master/packages/opentelemetry-cloud-monitoring-exporter).
+
+## Exporting measurements to Prometheus
+
+If you want to export your metrics to Prometheus you need to initialize Opentelemetry
+to use the Prometheus exporter `PrometheusExporter` which is included in the
+`@opentelemetry/exporter-prometheus`-package.
+
+```typescript
+const { PrometheusExporter } = require('@opentelemetry/exporter-prometheus');
+const { MeterProvider }  = require('@opentelemetry/sdk-metrics');
+
+// Add your port and startServer to the Prometheus options
+const options = { port: 9464, startServer: true };
+const exporter = new PrometheusExporter(options);
+
+// Creates MeterProvider and installs the exporter as a MetricReader
+const meterProvider = new MeterProvider();
+meterProvider.addMetricReader(exporter);
+const meter = meterProvider.getMeter('example-prometheus');
+
+// Now, start recording data
+const counter = meter.createCounter('metric_name', {
+  description: 'Example of a counter'
+});
+counter.add(10, { pid: process.pid });
+```
+
+In the above example the instantiated `PrometheusExporter` is configured to expose
+a new http server on port 9464. You can now access the metrics at the endpoint
+<http://localhost:9464/metrics>.  This is the url that can be scraped by Prometheus so it can consumed the metrics collected by Opentelemetry in your application.
+
+More information about Prometheus and how to configure can be found at:
+[https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config](Prometheus Scraping Config)
+
+For a fully functioning code example for using this exporter, please have a look
+at: <https://github.com/open-telemetry/opentelemetry-js/tree/main/experimental/examples/prometheus>
+
+## Exporting measurements to Opentelemetry Protocol
+
+Opentelemetry Javascript comes with three different kind of exporters that export
+the collected metrics in the Opentelemetry Protocol (OTLP). The three exports
+different in the transport method to send the metrics to a backend that supports
+the OTLP protocol, (a) [https://github.com/open-telemetry/opentelemetry-js/tree/main/experimental/packages/opentelemetry-exporter-metrics-otlp-http](over HTTP) (b) [https://www.npmjs.com/package/@opentelemetry/exporter-metrics-otlp-grpc](over GRPC) (c) [https://www.npmjs.com/package/@opentelemetry/exporter-metrics-otlp-proto](over Protofbuf).
```

Comment:
Switched up the order of recommendation a bit. :slightly_smiling_face:
I think http/proto should be the most supported by backends as recommended by the spec :slightly_smiling_face:  

```suggestion
the collected metrics in the OpenTelemetry Protocol (OTLP). Each exporter uses
a different transport method to send the metrics to a backend that supports
the OTLP protocol, (a) [https://www.npmjs.com/package/@opentelemetry/exporter-metrics-otlp-proto](over HTTP using Protofbuf) (b) [https://www.npmjs.com/package/@opentelemetry/exporter-metrics-otlp-grpc](over gRPC) (c) [https://github.com/open-telemetry/opentelemetry-js/tree/main/experimental/packages/opentelemetry-exporter-metrics-otlp-http](over HTTP using JSON).
```